### PR TITLE
Add true scale/offset to output

### PR DESCRIPTION
**Pull request by abellgithub**

### CODE REVIEW

**Comment by wonder-sk** [comment]
Looks good to me - it fixes indexing of the test dataset - and it seems that it also fixes precision loss of coordinates that has been happening with untwine.

File: epf/Epf.cpp [hunk 1]
```diff
@@ -226,10 +239,21 @@ PointCount Epf::createFileInfo(const StringList& input, StringList dimNames,
...
 
         QuickInfo qi = s->preview();
-
         if (!qi.valid())
             throw "Couldn't get quick info for '" + filename + "'.";
 
+        // Get scale values from the reader if they exist.
+        pdal::MetadataNode root = s->getMetadata();
+        pdal::MetadataNode m = root.findChild("scale_x");
+        if (m.valid())
```
Comment (connormanning) [inline epf/Epf.cpp:248]: Shouldn't this look like:
```
if (m_b.scale[0] == -1) m_b.scale[0] = m.value<double>();
else m_b.scale[0] = std::min(m_b.scale[0], m.value<double>());
```

Wouldn't we override a more precise scale like `0.001` with later occurring `0.01` by using `max()` as written, losing precision in the output?

Comment (abellgithub) [reply epf/Epf.cpp:248]: Yes. That's the idea.

Comment (connormanning) [reply epf/Epf.cpp:248]: So how do I index a dataset containing scales of both `0.001` and `0.01` losslessly?  Does it need to be set manually - I'm not seeing that option.  This seems backwards to me.

Comment (abellgithub) [reply epf/Epf.cpp:248]: This current use case is visualization through QGIS, not lossless storage. You have a choice, modify the scaling or fail.

Comment (connormanning) [reply epf/Epf.cpp:248]: That's kind of my point though - clipping a scale of 0.001 to 0.01 can result in very noticeable side-effects for visualization.  And regarding "modify the scaling", that's why I was asking about a manual option, how do I do this?  I don't see an option in untwine.cpp.

**Pull request merged**